sdcm/nemesis.py

```diff
@@ -255,11 +255,14 @@ def disrupt_stop_start_scylla_server(self):  # pylint: disable=invalid-name
     # This nemesis should be run with "private" ip_ssh_connections till the issue #665 is not fixed
     def disrupt_restart_then_repair_node(self):  # pylint: disable=invalid-name
         self._set_current_disruption('RestartThenRepairNode %s' % self.target_node)
-        self.target_node.restart()
-        self.log.info('Waiting scylla services to start after node restart')
-        self.target_node.wait_db_up(timeout=28800)  # 8 hours
-        self.log.info('Waiting JMX services to start after node restart')
-        self.target_node.wait_jmx_up()
+        # Task https://trello.com/c/llRuLIOJ/2110-add-dbeventfilter-for-nosuchcolumnfamily-error
+        # If this error happens during the first boot with the missing disk this issue is expected and it's not an issue
+        with DbEventsFilter(type='DatabaseLogEvent', line="Can't find a column family with UUID", node=self.target_node):
```

Comment:
Is this the only event we get there?
I think there are more events that we can filter there.

Reply:
Ok. So we need to decide which events should be filtered

Reply:
All error events during the first boot of the cluster.

Reply:
We agreed to merge it as is and update list of errors when received